HealthKitReporter/Classes/Service/HealthKitReader.swift

```diff
@@ -645,6 +649,41 @@ public class HealthKitReader {
         }
         return query
     }
+    /**
+     Queries objects (with anchors).
+     - Parameter type: **SampleType** types
+     - Parameter predicate: **NSPredicate** predicate (otpional). allSamples by default
+     - Parameter anchor: **HKQueryAnchor** anchor. HKAnchoredObjectQueryNoAnchor by default
+     - Parameter limit: **Int** anchor. HKObjectQueryNoLimit by default
+     - Parameter monitorUpdates: **Bool** set true to monitor updates. False by default.
+     - Parameter completionHandler: returns a block with samples
+     - Throws: HealthKitError.invalidType
+     */
+    public func anchoredObjectQuery(
```

Comment:
You have extended the anchoredObjectQuery with deleted objetcs and etc. Thank you, I think we can delete then this function since it is redundant and we have a full one at line 594

Reply:
I thought you or someone might be using this function, so to avoid breaking the existing functionality, I introduced the new function.
But as you said we can delete the redundant function